runtime/convertValues.go

```diff
@@ -256,36 +267,118 @@ func exportCompositeValue(
 	)
 }
 
-func exportDictionaryValue(
-	v *interpreter.DictionaryValue,
+func exportSimpleCompositeValue(
```

Comment:
The diff is misleading here, the code was not moved, it was added, but Git interprets it as a change